apps/mull-api/src/app/channel/inputs/channel.input.ts

```diff
@@ -8,10 +10,17 @@ export class ChannelInput implements Partial<Channel> {
 }
 
 @InputType()
-export class CreateChannelInput implements Partial<Channel> {
+export class CreateEventChannelInput implements Partial<EventChannel> {
   @Field()
   name: string;
 
   @Field()
   rights: number;
 }
+
+@InputType()
+export class CreateDmChannelInput implements Partial<DirectMessageChannel> {
+  @Field(() => [UserInput])
+  @ArrayMaxSize(2)
```

Comment:
We should also at add a Min here. I can create a DM channel with 0 or 1 participant